recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,19 @@
+cuda_compiler_version:         # [linux64]
+  - None                       # [linux64]
+  - 9.2                        # [linux64]
+  - 10.0                       # [linux64]
+  - 10.1                       # [linux64]
+  - 10.2                       # [linux64]
+  - 11.0                       # [linux64]
+
+docker_image:                                   # [linux]
+  - condaforge/linux-anvil-comp7                # [linux64]
+  - condaforge/linux-anvil-cuda:9.2             # [linux64]
+  - condaforge/linux-anvil-cuda:10.0            # [linux64]
+  - condaforge/linux-anvil-cuda:10.1            # [linux64]
+  - condaforge/linux-anvil-cuda:10.2            # [linux64]
+  - condaforge/linux-anvil-cuda:11.0            # [linux64]
+
+  - condaforge/linux-anvil-aarch64              # [aarch64]
+  - condaforge/linux-anvil-ppc64le              # [ppc64le]
```

Comment:
Aren't there different ppc64le cuda versions?

Reply:
I took these from [pinnings]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/53a32332a1553eb37b3edf4bf96b3311314f2fba/recipe/conda_build_config.yaml#L126-L127 ). Though they are [not being used yet]( https://github.com/conda-forge/nvcc-feedstock/blob/f2ee3a4798aaaa95680f55113026dd7a06f69478/recipe/meta.yaml#L9 ).

Admittedly we might want to add builds for other architectures, but I would rather save that for a future PR.

Reply:
Yeah, waiting makes sense